### DISCUSSION TITLE
2024/09/08/20240908_updatelog/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#27] [General]
# 2024/09/08/20240908_updatelog/

最近碎碎念有点多。 碎碎念比较多，可能是因为这段时间以来生活一直不算太顺心，且极其缺乏宣泄渠道，只能把一些情绪藏在日记里。一个人宅在实验室里的时候啥也不想做，只想刷知乎刷b站，出门转一圈又要被大街上成双成对的情侣或者成群结队的人群给羡慕到（也许是气死），生完闷气继续宅继续刷知乎刷b站，一点改变都没有。

https://wz.anoms.top/2024/09/08/20240908_updatelog/

**Comment by Hazelandinstant** [comment]
我有个观点不知当讲不当讲（🐶）你好闲。当我整天准备各种结果材料分不清外边是黑天白天的时候，竟然有人在这伤春悲秋。既然有大把时光好好学习不行吗？英语学否？文章发否？读书否？运动否？探索新世界否？

**Reply by cyclinbox** [reply]
感谢佳桢老师来指正我过于闲的错误🐶
以及前段时间你整天准备材料到昏天黑地，努力的结果毕竟得到了回报。特别佩服你身上的拼劲儿，以及祝贺最终的结果。如果你愿意的话，或许也可以享受闲下来的时光（但是，最好别学我，做学问还是卷一点比较好）
“伤春悲秋”更多是在察觉到情绪涌现，一段时间以来积攒下一篇文章。可能我属于那种比较佛系的人吧（低情商：自觉性不强），外界影响下很容易忘记该干什么，平时也确实浪费了一些时间。不过，必须为自己正名，虽然，文章发，否；探索新世界，否；但是，读书学习练英语，这个不能否（而且空闲时间看了不少书）；运动，这个也不能否，不然对不起每天健身房的3km。
一起加油呀~

**Comment by Shall-We-Dance** [comment]
哇我刚才歌单循环到《捞月亮的人》给我听emo了，去年这时候我在北京的时候经常单曲循环这首歌，“淚光裝飾夜晚 路燈點綴感嘆”。感觉真的好容易悲秋。
来美国后我觉得建立社交关系很累，这里的人活得更像是一个个独立的原子，我不需要和任何人社交就可以活下去。。。就是经常一天到晚都没有人说话，b站也刷得多，怎么说呢，经常会有像“霧色安撫月缺 大街依舊積雪 甚麼心事也許不必說”的感觉。

**Reply by cyclinbox** [reply]
🫂🫂🫂 
或许这就是越长大越孤独（？）

我发现，最热闹最温暖的时光还是大学时期；毕业之后，大家的边界感越来越强，好像也越来越难在新环境中建立起深层次的社交关系。美国社会可能更强调个人独立，这使得在这种环境下更容易感到孤独。

之前读过余光中的散文集《此心安处便是吾乡》，其中流露出的淡淡乡愁与现在的季节氛围特别契合。“自古逢秋悲寂寥”，秋天的凉意似乎也加深了这份孤独感。不过，生活总是在继续，愿我们大家都能在这个季节里找到属于自己的温暖。

（PS：好奇美国的生活到底是咋样呀，能在评论区说说嘛🐶）

**Reply by Shall-We-Dance** [reply]
挺好的，我感觉真实的美元:人民币购买力差不多2:1，这边日常消费的价格差不多就是（国内相似服务人民币价格x2）美元。平时吃饭差不多10-20刀，电子产品等折扣还是能比国内便宜，平时大多都比国内贵。而且涉及人工的费用都很贵。我现在不急的东西都是淘宝买了然后转运过来。学校发的工资足够租房+养车，每个月能稍微存一点下来，这个还是不错的。美国越大的城市越乱，越村越舒服。网络不太行，我自己家的宽带50刀一个月，300Mbps下行10Mbps上行，路由表很差，到纽约市延迟都有20ms。学校的网络很快但是IPv6普及很差。

**Reply by cyclinbox** [reply]
这么看来美国的生活真的很不错，令人羡慕。
